build/ci/clean_demo_pr.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export failed_k8s_checks=false
+export failed_openshift_checks=false
```

Comment:
consider using `passed` instead of `failed`, so that if `fail` (asserted in `grep -i fail`) ever changes, we won't have a false positive

Reply:
done